### PR TITLE
Feature/customtheme param

### PR DESCRIPTION
**Pull request by CAMOBAP**
- #10 

Notes:
 - opposite to iOS implementation we support only JSON string here (JS Object string will be failed)
 - I have added one more error `INVALID_CUSTOM_THEME` with code `32`

Open question:
 - should we customize `loaderContainer` background based on `theme` or `customTheme`? For now we just use `@android:color/white` which is `white`/`light` for `light` system theme and `dark` for `dark` system theme, so it doesn't depend on `theme` or `customTheme` params

### CODE REVIEW

**Comment by e271828-** [comment]
(Don't forget the api.js change required for customTheme.)

**Comment by CAMOBAP** [comment]
> (Don't forget the api.js change required for customTheme.)

good catch, I will update PR

**Comment by CAMOBAP** [comment]
@e271828- done, could you please check Notes and Open questions in the top comment

File: README.md [hunk 1]
```diff
@@ -90,6 +90,7 @@ HCaptcha.getClient(this).verifyWithHCaptcha(config)...;
 |`reportapi`|String|No|-|See Enterprise docs.|
 |`assethost`|String|No|-|See Enterprise docs.|
 |`imghost`|String|No|-|See Enterprise docs.|
+|`customTheme`|String|No|-|See Enterprise docs.|
```
Comment (e271828-) [inline README.md:93]:
```suggestion
|`customTheme`|Stringified JSON|No|-|See Enterprise docs.|
```

**Comment by e271828-** [comment]
> should we customize loaderContainer background based on theme or customTheme? For now we just use @android:color/white which is white/light for light system theme and dark for dark system theme, so it doesn't depend on theme or customTheme params

Think it's fine to leave as is for now. We'll see if anyone makes a request for this, but I'd expect them to just turn off loader.

**Review by e271828- — APPROVED** [review]

**Pull request merged**